lib/Doctrine/ORM/Tools/Pagination/Paginator.php

```diff
@@ -265,4 +268,23 @@ private function unbindUnusedQueryParams(Query $query): void
 
         $query->setParameters($parameters);
     }
+
+    /**
+     * @param mixed[] $identifiers
+     *
+     * @return mixed[]
+     */
+    private function convertWhereInIdentifiersToDatabaseValue(array $identifiers): array
```

Comment:
Unless I'm missing something this should be plural.

```suggestion
    private function convertWhereInIdentifiersToDatabaseValues(array $identifiers): array
```

Reply:
fixed